tests/resolve_select_expression.rs

```diff
@@ -273,6 +274,7 @@ use-bar =
 }
 
 #[test]
+#[ignore]
```

Comment:
Why do you need to ignore this test?

Reply:
is this and the other tests ignored because there's currently no way to retrieve terms from `MessageContext`?

Reply:
yup.